heartbeat/docker-compose

```diff
@@ -185,19 +196,25 @@ docker_compose_status()
 
 docker_compose_start()
 {
+
+	sleep 5
+
 	docker_compose_validate_all
 	docker_compose_status >/dev/null 2>&1
 	retVal=$?
 	# return success if docker service is running
 	[ $retVal -eq $OCF_SUCCESS ] && exit $OCF_SUCCESS
 
+	sleep 10
```

Comment:
If you need to wait here you need to add a loop checking every 1-2s (and fails when Pacemaker timeouts) to avoid waiting too much in general, and also work for cases where it takes more than 10s.

Reply:
yes, agree. I am still working on it... it could be improved in the future version.